tests/Test/AbstractFixerTestCase.php

```diff
@@ -669,7 +668,7 @@ protected static function assertCorrectCasing(string $haystack, string $needle,
     }
 
     /**
-     * @return \ReflectionClass<FixerInterface>
+     * @return \ReflectionClass<AbstractFixer>
```

Comment:
AbstractFixer is internal. I would prefer to keep FixerInterface here.

Reply:
> I would prefer to keep FixerInterface here.

No, you wouldn't 😁 (click the picture to see the source):

[![image](https://github.com/user-attachments/assets/eb7641ad-d80a-491a-9a9a-a19c9952d3ce)](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.68.1/tests/Test/AbstractFixerTestCase.php#L101-L103)

Jokes aside, we can update it the other way: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8390/commits/92c7844b2d795b620b6421d5d5976af8e7c3c44b